.github/workflows/technology-release.yml

```diff
@@ -0,0 +1,111 @@
+name: Release
+
+on:
+  workflow_call:
+    inputs:
+      branch:
+        required: true
+        type: string
+
+jobs:
+  Determine-Version:
+    runs-on: ubuntu-latest
+    outputs:
+      VERSION: ${{ steps.get-version.outputs.VERSION }}
+    steps:
+      - name: "Get version"
+        id: get-version
+        run: |
+          echo "VERSION=$(echo ${{ inputs.branch }} | awk -F'/' '{print $2}')" >> $GITHUB_OUTPUT
+
+  Publish-Artefacts:
+    runs-on: ubuntu-latest
+    needs: [ Determine-Version ]
+    if: needs.Secrets-Presence.outputs.HAS_OSSRH
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          ref: ${{ inputs.branch }}
+
+      - uses: eclipse-edc/.github/.github/actions/setup-build@main
+
+      # Import GPG Key
+      - uses: eclipse-edc/.github/.github/actions/import-gpg-key@main
+        if: needs.Secrets-Presence.outputs.HAS_OSSRH
+        name: "Import GPG Key"
+        with:
+          gpg-private-key: ${{ secrets.ORG_GPG_PRIVATE_KEY }}
+      - name: "Publish To OSSRH/MavenCentral"
+        if: needs.Secrets-Presence.outputs.HAS_OSSRH
+        env:
+          OSSRH_PASSWORD: ${{ secrets.ORG_OSSRH_PASSWORD }}
+          OSSRH_USER: ${{ secrets.ORG_OSSRH_USERNAME }}
+          VERSION: ${{ needs.Determine-Version.outputs.VERSION }}
+        run: |-
+          echo "Publishing Version $VERSION to Sonatype"
+          ./gradlew publishToSonatype closeAndReleaseSonatypeStagingRepository --no-parallel -Pversion=$VERSION -Psigning.gnupg.executable=gpg -Psigning.gnupg.passphrase="${{ secrets.ORG_GPG_PASSPHRASE }}"
+
+  Create-Tag:
+    needs: [ Publish-Artefacts ]
+    runs-on: ubuntu-latest
+    outputs:
+      TAG: ${{ steps.set-tag.outputs.TAG }}
+    steps:
+      - shell: bash
+        id: set-tag
+        run: |
+          git config user.name "eclipse-edc-bot"
+          git config user.email "edc-bot@eclipse.org"
+          
+          TAG=v${{ needs.Determine-Version.outputs.VERSION }}
+          
+          git tag $TAG
+          git push -f origin tag $TAG
```

Comment:
here, `-f` is potentially harmful. What would be the use case of needing to overwrite tags?

Reply:
same here, in case of eventual failure, but, true, at this point artifacts would have been already published so there's no real need for this `force`